docker-compose.yml

```diff
@@ -4,9 +4,14 @@ services:
   db:
     image: postgres:10
     ports:
-      - "5432:5432"
+      # We changed the default port so people having already a postgres instance running on port 5432 won't have a port conflict
+      - "5433:5432"
     volumes:
       - ./postgres-data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=postgres
```

Comment:
POSTGRES_PASSWORD is mandatory from the doc : https://hub.docker.com/_/postgres
POSTGRES_USER and POSTGRES_DB make the postgres config explicit so it easier for people to understand